pandas/tests/io/test_pytables.py

```diff
@@ -2192,9 +2192,9 @@ def test_unimplemented_dtypes_table_columns(self):
             pytest.raises(TypeError, store.append, 'df_unimplemented', df)
 
     @pytest.mark.skipif(
-        not _np_version_under1p15,
-        reason=("pytables conda build package needs build "
-                "with numpy 1.15: gh-22098"))
+        LooseVersion(np.__version__) == LooseVersion('1.15'),
```

Comment:
I see LooseVersion('1.15') and LooseVersion('1.15.0') are not equal. But,  StrictVersion('1.15') and Strict Version('1.15.0') are equal. We can use StrictVersion, in that we can use '1.15' or '1.15.0'. If we are using LooseVersion, then it should be '1.15.0'.

Reply:
I have made changes with LooseVersion('1.15.0'). I think this will suffice. Please let me know if we have to use StrictVersion instead @jreback @TomAugspurger ?